classes/abicheck.bbclass

```diff
@@ -49,6 +49,22 @@ python binary_audit_abixml_compare_to_ref() {
     t0 = time.monotonic()
 
     pn = d.getVar("PN")
+    
+    util.note("FILESPATH {}".format(d.getVar("FILESPATH")))
+    
+    util.note("WORKDIR {}".format(d.getVar("WORKDIR")))
+    recipe_suppr = d.getVar("WORKDIR") + "/*_recipe.suppr"
+    
+    suppr = glob.glob(recipe_suppr)
+    util.note("RECIPE SUPPRESSION: ".format(str(suppr)))
```

Comment:
This will be contained in the end version of the array, probably no need to log this separately.

Thanks